docs/guide/reactive-progressive.md

```diff
@@ -0,0 +1,205 @@
+# Use State Management Progressive
+
+> If you want to use whole Goldfish state Management, please visit [Use state Management](./reactive.md).
+
+## Alipay mini program State Management
+
+The original framework of Alipay mini program provides the Page/Component level data state management ([this.setData()](https://docs.alipay.com/mini/framework/page-detail#pageprototypesetdatadata-object-callback-function)), but that is not available for Page or Component across scenarios. The general solution is to put data into globalData to solve the data sharing problem across pages.
+
+It can be seen that basic mechanism is available for state management in the original Alipay mini program, which does not suffice the state management in complicated projects.
+
+## Goldfish Reactive State Management
+
+On basis of the related great ideas in the community, Goldfish deposits a set of mini program friendly responsive state management framework, which enables the user to flexibly and efficiently handle the state data.
+
+In Goldfish, state data is responsive, you can use `observable`, `state`, `computed` to create reactive data, and use `AppStore` `PageStore` `ComponentStore` declare store, finally use `createApp` `createPage` `createComponent` to connect store data.
```

Comment:
responsive -> reactive

Reply:
finally -> then

Reply:
to connect store data -> to connect the store with the view instance (Page, Component, or App)